pysatNASA/instruments/formosat1_ivm.py

```diff
@@ -1,4 +1,12 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
 """Module for the Formosat-1 IVM instrument.
```

Comment:
From a user perspective, unfortunate side-effect of funding patterns that formosat-x satellite data is spread across multiple packages. I don't have a solution in this comment box but it would be easier on users if there was but one place, or specific user directions. Perhaps a telecom topic.

Reply:
Add it to the agenda